buildMessage.js

```diff
@@ -62,19 +76,48 @@ function profile (evt) {
 }
 
 function processBlocked (evt) {
-  const trace = evt.stacktrace.replace('```', `'''`)
-  const lines = trace.split('\n')
-  const first = lines.shift()
-  const result = '> ' + first + ' ```' + lines.join('\n') + '```'
-  return result
+  if (evt.stacktrace) {
+    const trace = evt.stacktrace.replace('```', `'''`)
+    const lines = trace.split('\n')
+    const first = lines.shift()
+    const result = '> ' + first + ' ```' + lines.join('\n') + '```'
+    return result
+  } else if (evt.stack) {
+    const trace = evt.stack.replace('```', `'''`)
+    const lines = trace.split('\n')
```

Comment:
This code is redundant I will use ternary to assign trace and remove the duplicated code:

```javascript
const trace = (evt.stacktrace) ? evt.stacktrace.replace('```', `'''`) : evt.stack.replace('```', `'''`)
```